### DISCUSSION TITLE
is there any method how to switch proxy while using the driver after it gets instanciated ?

### DISCUSSION THREAD

**Discussion by Black-Hole1004** [seleniumbase/SeleniumBase#2723] [Q&A] [answered]
I want to use a proxy when instantiating the object `Driver(uc=True, headed=True, proxy=proxy)`, and then switch to another proxy at some point, I have found older discussion but it uses SB, is there any method with Driver ?

**Comment by mdmintz** [comment, marked as answer]
To switch the proxy in the middle of a test, you can use `wire=True` with `driver.set_wire_proxy(string)`.
But note that Wire Mode cannot be used together with UC Mode.

More info: https://github.com/seleniumbase/SeleniumBase/issues/2145

**Comment by Black-Hole1004** [comment]
`AttributeError: 'Chrome' object has no attribute 'set_wire_proxy'`, tried this `with SB(uc=True, wire=True) as sb:
    driver = sb.driver
    driver.set_wire_proxy(proxy)`, and this `driver = Driver(uc=True, wire=True)
driver.set_wire_proxy(proxy)`, both doesn't have the set_wire_proxy method

**Reply by mdmintz** [reply]
As I mentioned before, you cannot combine Wire Mode with UC Mode. Pick one or the other.
